SiriIntents/IntentHandler.m

```diff
@@ -330,7 +330,7 @@ - (void)resolveContacts:(nullable NSArray<INPerson *> *)contacts withCompletion:
                         if (userRoomSummaries)
                             [userRoomSummaries addObject:summary];
                         else
-                            roomSummaries[diretUserId] = [NSMutableArray arrayWithObject:summary];
+                            roomSummaries[diretUserId] = [@[summary] mutableCopy];
```

Comment:
JFYI, your changes leads for allocation of 2 objects instead of one as it was in the previous revision.

The first object is instance of `NSArray` class created via array literal syntax and the second one is instance of `NSMutableArray` which is a result of `mutableCopy` method call.

Reply:
Thanks for the heads up, I reverted that :)